src/convertToObject.js

```diff
@@ -7,6 +7,20 @@
  */
 function convertToObject(sourceString) {
   // write your code here
+  const result = {};
+
+  const x = sourceString
+    .toString()
+    .split(';')
+    .map((item) => item.toString().split(':'));
```

Comment:
[CODE KNOWLEDGE] - There's no need to call 'toString()' on 'sourceString' because it is already a string. Additionally, 'map' does not mutate the original array, so you don't need to call 'toString()' on each 'item' before splitting it.